packages/dates/CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 <!-- Unreleased changes should go to UNRELEASED.md -->
 
----
+## Memoize `Intl.DateTimeFormat` [#643](https://github.com/Shopify/quilt/pull/643)
```

Comment:
I don't think this would be a header?

I think the header would either be `Added`, `Changed`, or `Fixed`. Hard to tell in this case. https://keepachangelog.com/en/1.0.0/#how

Reply:
Maybe use `Changed formatDate to use memoized getDateTimeFormat()`.

Reply:
Added to `UNRELEASED.md`
https://github.com/Shopify/quilt/pull/643/files#diff-ffa836aa672cf05f1ca30747cd245f92